fleetctl/fleetctl.go

```diff
@@ -75,10 +75,10 @@ var (
 
 	// flags used by all commands
 	globalFlags = struct {
-		Debug   bool
-		Version bool
-		Help    bool
-		currentCommand	string
+		Debug          bool
+		Version        bool
+		Help           bool
+		currentCommand string
```

Comment:
Don't include unrelated whitespace changes in code you do not touch in this patch -- this should go into the previous commit.

Except that in this case I don't actually see any point in having these as separate commits at all... And in fact that's true for almost(?) all commits in this PR: they don't really make any sense in isolation -- so they should just be squashed into a single one.

Separate commits only make sense if you have distinct bits of functionality, or if you have functionality-preserving changes to other parts of code that your main changes depend upon. I don't see that here: all these changes are inherently connected. (In fact I'm pretty sure some of the intermediate commits here result in nonsensical behaviour, and probably wouldn't pass the test suite...)